youtube_dl/extractor/audioboom.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class AudioBoomIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?audioboom\.com/boos/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://audioboom.com/boos/4233286-ai-safety-and-the-legacy-of-bletchley-park',
+        'md5': 'fabd6bf64d9c8df183e8db5963963375',
+        'info_dict': {
+            'id': '4233286',
+            'ext': 'mp3',
+            'title': 'AI Safety and The Legacy of Bletchley Park',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._og_search_title(webpage)
+        _id = self._search_regex(r'https?://(?:www\.)?audioboom\.com/boos/(?P<id>[0-9]+)', url, 'id')
```

Comment:
This matches just any plain text that meet this pattern. Instead it should extract it via pattern that has embed semantics. This also won't work if actual media is stored under different URI.